### PR TITLE
Fixed timedelta issue and inprogress update docker file issue.

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #, if available:*

*Description of changes:*
1. Fixed timedelta issue;
2. Update inprogress update job' docker file to use 64bit Oracle libraries;
3. Fixed typos;

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: api/namex/services/statistics/wait_time_statistics.py [hunk 1]
```diff
@@ -33,22 +34,24 @@ def get_waiting_time_regular_queue(cls, unit):
...
-        else:
-            oldest_draft_date = oldest_draft.submittedDate
-
-        # add one to waiting time to account for current day
-        delta = submitted_date - oldest_draft_date
-        response_data = {'oldest_draft': oldest_draft_date.isoformat(), 'waiting_time': delta.days}
+    def get_waiting_time_dict(cls):
+        try:
+            oldest_draft = Request.get_oldest_draft()
+            if oldest_draft is None:
```
Comment (thorwolpert) [inline api/namex/services/statistics/wait_time_statistics.py:45]: Consider the walrus :=
```
if not (oldest_draft := Request.get_oldest_draft()):
```
or
```
if oldest_draft := Request.get_oldest_draft():
    oldest_draft_date = oldest_draft.submittedDate
else:
    oldest_draft_date = datetime.now().astimezone()
```

**Review by thorwolpert — COMMENTED** [review]
just one suggestion

File: api/namex/services/statistics/wait_time_statistics.py (hunk 1 above)
Comment (pwei1018) [reply api/namex/services/statistics/wait_time_statistics.py]: I changed to use Walrus Operator.

**Review by thorwolpert — APPROVED** [review]
linting & tests will fail in NameX for a while.
i'm ok with leaving it in the CI, as it needs to get fixed anyways.
could we switch linting to WARN instead of FAIL so that at least the tests are run?

**Review by thorwolpert — APPROVED** [review]
thank you

**Review by kialj876 — APPROVED** [review]

**Pull request merged**